src/extension/inlineEdits/vscode-node/features/diagnosticsBasedCompletions/diagnosticsCompletions.ts

```diff
@@ -288,30 +286,20 @@ export class CodeAction {
 			action.diagnostics?.map(diagnostic => Diagnostic.fromVSCodeDiagnostic(diagnostic)) ?? [],
 			action.edit,
 			action.command,
-			action.kind,
-			action.isPreferred,
-			action.disabled
 		);
 	}
 
 	private constructor(
 		public readonly title: string,
 		public readonly diagnostics: Diagnostic[],
 		private readonly edit?: vscode.WorkspaceEdit,
-		public readonly command?: vscode.Command,
-		protected readonly kind?: vscode.CodeActionKind,
```

Comment:
These didn't need to be public
& remove other items that were not used (e.g. Kind and the like, as it would be more work to create non-vscode classes that map to those, when its not used)